### ISSUE TITLE
Supported Use Cases

### ISSUE THREAD

**Issue by danielwilms** [w3c/vsso#6]
Part of the introduction should be to clarify the supported use cases. One proposal would be to have three supported use case cluster:

* **Analytics: Current Value**: This cluster includes all use-cases where we want to work on the current state of the vehicle (e.g. give me all cars of a specific model, notify when the doors are closed but the windows are open). The time domain, in a sense of the behaviour over time plays a less important role.
* **Analytics: Behaviour over time**: Here the time aspect is more important and I would see VSSo as a domain ontology for e.g. SSN/SOSA, IoTStream. Imagine a situation, where you want to analyse a sequence of historic events, or incoming data streams (e.g. how did the speed change over time? What is the context of a braking event?).
* **Service description**: Defines the service interfaces, independently of the underlying protocol, so that it can be mapped to one during deployment. Part of it is service discovery. In this case VSSo would serve as domain ontology for the Web Of Things Thing Description.

Any opinions?

**Comment by danielwilms** [comment]
@klotzbenjamin: additional domains interaction (Smart City, Charging, etc.)

When and how to reach out? @tguild reaching out to other domains, invites into meeting, etc., but mostly for awareness.

**Comment by danielwilms** [comment]
Thinking about it a bit longer I think, that maybe the additional interactions with other domains is not a separate use case but rather part of the use cases. Thinking about Smart City, it most likely falls into use case cluster 1, consuming the current value of the vehicle. Charging in use case cluster 3, where you want to interact with the vehicle. We could mention the examples to describe the use cases better.
What do you think @klotzbenjamin, @tguild?

Another suggestion, maybe the first one could be renamed into `Digital Twin` and discussed accordingly.

**Comment by klotzbenjamin** [comment]
> Thinking about it a bit longer I think, that maybe the additional interactions with other domains is not a separate use case but rather part of the use cases. Thinking about Smart City, it most likely falls into use case cluster 1, consuming the current value of the vehicle. Charging in use case cluster 3, where you want to interact with the vehicle. We could mention the examples to describe the use cases better.

I agree, mostly with the cluster 3 case: any external domain can interact with through a domain-independent ontology like the WoT. More precisely I think it is the core interest of service descriptions.

> Another suggestion, maybe the first one could be renamed into `Digital Twin` and discussed accordingly.

+1, It is more accurate than "Analytics"

**Comment by tguild** [comment]
I agree and some aspects of eg SmartCities span all three clusters. Urban traffic management would benefit from trends of aggregate data - second case.

Although buzzwords give me pause, agree it is much more than just enabling analytics.

**Comment by anntexx** [comment]
What about the following use cases regarding Smart City:

Finding a parking space: In today’s cities, searching for a free parking lot takes an average of 30 minutes or about 4.5km drive each time [1]. Knowing the positions of parking spaces as well as their occupancy could reduce these efforts significantly. 
An ontology based system can assist in discovering the next available parking lot by linking information from smart parking sensors as well as including additional information (e.g., dimension of vehicle, charging options available, etc.). 

[1] https://www.bosch.com/explore-and-experience/communitybased-parking/
Some more ideas published in a paper back in 2018: https://ieeexplore.ieee.org/abstract/document/8417783

**Comment by tguild** [comment]
For both analytics use cases it might be worth adding AI for AV as I have seen one presentation on that use case. Behavior over time for machine learning and Current value for realtime decision making.
